app/views/public_recipes/index.html.erb

```diff
@@ -0,0 +1,8 @@
+<% @public_recipes.each do |recipe| %>
+  <div class="recipe border d-flex flex-row justify-content-center align-items-center">
+    <h2 style="margin: 30px"><%= recipe.name %></h2>
+    <p style="margin: 30px">By <%= User.find(recipe.user_id).name %></p>
+    <p style="margin: 30px">Total food items: <%= recipe.foods.count %></p>
+    <p style="margin: 30px">Total price: <%= recipe.foods.sum { |food| food.quantity * food.price } %></p>
```

Comment:
- [ ] Based on the requirements of this project, clicking on a recipe item should lead to the recipe details page. Right now there's no link to click on it.
- [ ] As stated in the requirements, let's make sure to add the recipe detail page.